### PR TITLE
Add note that cuda quantization is not supported

### PR DESCRIPTION
**Pull request by dzhulgakov**
People get confused with partial support otherwise: #27811 #27729

Suggestions on where else put warnings are welcomed (probably in tutorials - cc @SethHWeidman )

### CODE REVIEW

**Comment by dongfangduoshou123** [comment]
> People get confused with partial support otherwise: #27811 #27729
> 
> Suggestions on where else put warnings are welcomed (probably in tutorials - cc @SethHWeidman )



In the future, are there any plans to support CUDA quantification in pytorch？

File: docs/source/quantization.rst [hunk 1]
```diff
@@ -26,6 +26,20 @@ perform all or part of the computation in lower precision. Higher-level APIs are
 provided that incorporate typical workflows of converting FP32 model to lower
 precision with minimal accuracy loss.
 
+Today, PyTorch supports the following backends for running quantized operators efficiently:
+
+* x86 CPUs with AVX2 support or higher (without AVX2 some operations have inefficient implementations)
+* ARM CPUs (typically found in mobile/embedded devices)
+
+Corresponding implementation is chosen automatically based on the PyTorch build mode.
```
Comment (soumith) [inline docs/source/quantization.rst]: The corresponding implementation

**Review by soumith — APPROVED** [review]

**Comment by raghuramank100** [comment]
We will also update the tutorial documentation to reflect this.

**Review by gottbrath — APPROVED** [review]
Looks good to me! Thanks!

**Pull request closed without merging**